# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../suspend/HP-2133.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- suspend/whitelist.c.new	2009-02-19 10:23:26.000000000 +0000
+++ suspend/whitelist.c	2009-02-19 10:26:19.000000000 +0000
@@ -215,6 +215,8 @@
 	{ "Hewlett-Packard ",		"Pavilion zt3300 *",	"",	"68BAL*", S3_BIOS },
 	/* A desktop with integrated graphics, Bo Thysell <thysell@fusemail.com> */
 	{ "HP Pavilion 061",		"EJ192AA-ABS t3250.se",	"",	"", 0 },
+	/* Rene Rebe <rene@exactcode.de>, VIA Chrome 9 */
+	{ "Hewlett-Packard",  "HP 2133", "", "", VBE_POST },
 
 	/* R51 confirmed by Christian Zoz and Sitsofe Wheeler <sitsofe@yahoo.com> */
 	{ "IBM",			"",		"ThinkPad R51",	"", 0 },
